docs/sources/reference/components/prometheus/prometheus.write.queue.md

```diff
@@ -83,16 +83,17 @@ The `endpoint` block describes a single location to send metrics to. Multiple
 
 The following arguments are supported:
 
-Name | Type | Description                                                      | Default | Required
----- | ---- |------------------------------------------------------------------| ------ | --------
-`url` | `string` | Full URL to send metrics to.                                     | | yes
-`write_timeout` | `duration` | Timeout for requests made to the URL.                            | `"30s"` | no
-`retry_backoff` | `duration` | How often to wait between retries.                               | `1s` | no
-`max_retry_attempts` | Maximum number of retries before dropping the batch. | `0`                                                              | no
-`batch_count` | `uint` | How many series to queue in each queue.                          | `1000` | no
-`flush_interval` | `duration` | How often to wait until sending if `batch_count` is not trigger. | `1s` | no
-`parallelism` | `uint` | How many parallel batches to write.                              | 10 | no
-`external_labels` | `map(string)` | Labels to add to metrics sent over the network.                  | | no
+Name | Type | Description                                                        | Default | Required
+---- | ---- |--------------------------------------------------------------------| ------ | --------
+`url` | `string` | Full URL to send metrics to.                                       | | yes
+`bearer_token` | `secret` | Bearer token to authenticate with.                                 |  | no
+`write_timeout` | `duration` | Timeout for requests made to the URL.                              | `"30s"` | no
+`retry_backoff` | `duration` | How often to wait between retries.                                 | `1s` | no
```

Comment:
How often or how long?

Reply:
@clayton-cornell I dont think he changed most of these, I think adding bearer_token triggered them to look different. I can create a separate PR for these doc comments related to backoff and flush interval.

Reply:
Yeah I didn't, it's my IDE that apparently tried to format it, only thing I changed is adding this line and also fixing the typo (trigger -> triggered) in another one.

Reply:
I might change it here as well though if you prefer

Reply:
Handling it in #1917